apps/frontline-widgets/src/formstyle.css

```
@@ -0,0 +1,238 @@
+  box-shadow: 0 3px 18px -2px rgba(0, 0, 0, 0.2);
+}
+
+/* dropdown */
+.erxes-dropdown-iframe {
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  animation-delay: 1s;
+  -webkit-animation-delay: 1s;
+  animation: fadeInDown;
```

Biome (2.3.14)

[error] Unexpected shorthand property animation after animation-delay



(lint/suspicious/noShorthandPropertyOverrides)